### ISSUE TITLE
Environmental/Social labels for items

### ISSUE THREAD

**Issue by idolum** [OP-TED/ePO#343]
To proof that an item or services fulfills environmental or social requirements with respect to producing/providing the item, the manufacturer can apply for an item. If they succeed, they an label is assigned to them, that can be used by the manufacturer that these requirement are fulfilled. This can be used by buyers to find items that fulfill certain  environmental or social requirements.

The following information on a label are usually required

* Name of the label that was given to an item. For instance, EU Ecolabel
* A standardized code that identifies the label that was given to an item. For instance, 
* A code or license number identifying the label instance given to an item. For instance, IT/044/005 for the EU Ecolabel given to the item
* A URL referring for the certificate document that proofs validity of the label given to the item. For instance, http://ec.europa.eu/ecat/product/en/25214/indoor-and-outdoor-paints-1--colorante-all-acqua-pe
* Name of the issuing party. For instance, the RAL gGmbH issues the EU Ecolabel

### User Story

A person in a public organisation has to buy new printing paper. Due to the procurement policy of the organisation, aspects of sustainability have to be considered. There the person search the catalogue for printing paper that have a EU Ecolabel, as this is the recognized by the the public organisation. 

The person finds the following item with the following information on the label

* Name: Eco Printing Paper
* Item Label
  - Name: EU Ecolabel
  - Code: Ecolabel
  - Label number: IT/001/0001
  - Type: Eco
  - Reference URI: http://ec.europa.eu/ecat/product/en/12345/sample-eco-printing-paper
* Item Label Issuer
  - Name: RAL gGmbh
 
### Proposal

Based in the CEN/WS BII3 data model the following is proposed:

![image](https://user-images.githubusercontent.com/1178743/174061714-19ae8ce4-ef1a-4300-9a72-140f11c984af.png)

Definitions:

* Label: A label proofing that certain requirements regarding social, ethical and quality aspects are respected by producing/providing the item.
  - hasTypeCode: The label type such environmental, quality, social etc.
  - hasReferenceUri: A reference to where the label specification can be found, e.g. a URI.
* Label Issuer: Agent issuing the party

**Comment by andreea-pasare** [comment]
@idolum, can you provide an update on this ticket, please?

**Comment by idolum** [comment]
@andreea-pasare Extended the ticket. My apologies for being late!

**Comment by andreea-pasare** [comment]
![image](https://user-images.githubusercontent.com/88081600/178715272-c3379a4b-cc09-4338-9087-82ca06c40d2a.png)

**Comment by giorgialodi** [comment]
What's the difference between Certification and Label? The URL property is on the label but in this thread there is indication that the URL is of the certification document. Is Certification a document?

**Comment by costezki** [comment]
After discussion in the workgroup, we have convened that the CCCEV model defines the appropriate abstractions for the "certification" and label "concepts". 
Moreover, it was clear that other use-cases exist for the provisioning evidence that a product respects certain requirements.  For example, what is specified in Directive 24, Article 43 (e.g. technical dossier, or technical tests ). These use cases, however, shall be treated separately elsewhere. 

@giorgialodi You are right, we have moved the URL on the "certification" class. 

![image](https://user-images.githubusercontent.com/13567133/179777692-02939f34-8440-4a5e-9f28-f6d27c4df8fa.png)
